experimental/veggieseasons/lib/widgets/search_bar.dart

```diff
@@ -36,29 +36,13 @@ class SearchBar extends StatelessWidget {
             ),
             child: Row(
               children: [
-                ExcludeSemantics(
-                  child: Icon(
-                    CupertinoIcons.search,
-                    color: Styles.searchIconColor,
-                  ),
-                ),
+                /// Updated custom search field to CupertinoSearchTextField
                 Expanded(
-                  child: CupertinoTextField(
+                  child: CupertinoSearchTextField(
                     controller: controller,
                     focusNode: focusNode,
                     decoration: null,
```

Comment:
It's by default a nullable type, so you can avoid passing null.

Reply:
Sure @AyushBherwani1998 I'll do the necessary changes and revert you back